js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,6 +1,6 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js');
+var ExtensionStore = require('./ExtensionStore.js').instance;
```

Comment:
@sophistifunk I'm all ears for a better way to do this, I would love to have some sort of DI to be able to hook this together rather than relying on `init()` method calls